regparser/tree/reg_text.py

```diff
@@ -84,6 +85,95 @@ def build_subpart(text, part):
         "", [], label, subpart_title, node_type=struct.Node.SUBPART)
 
 
+def subjgrp_label(starting_text, letter_list):
+    words = starting_text.split(" ")
+    candidate_text = ""
+    suffixes = [""] + [_ for _ in string.lowercase[1:]]
+    if len(words) == 1:
```

Comment:
This function has a single, very large `if-else`, which makes me think it should be broken up.

Somewhat related: this returns a pair of the text + the extended letter list. It might be less tightly coupled if it just returned the text (allowing the caller to extend the list, if desired).